src/sentry/middleware/stats.py

```diff
@@ -28,7 +28,7 @@ def process_exception(self, request, exception):
 
 class RequestTimingMiddleware(object):
     allowed_methods = ("POST", "GET")
-    allowed_paths = ("sentry.web.api", "sentry.api.endpoints")  # Store endpoints
+    allowed_paths = ("sentry.web.api", "sentry.api.endpoints", "getsentry.api", "getsentry.web")  # Store endpoints
```

Comment:
We shouldn't do this in sentry, it has no idea that getsentry exists. I think the ideal fix here is to make `allowed_paths` a Django setting, and then override it in getsentry

Reply:
cool.  I'll do that.  (Once I figure out how django settings work.  :D )

Reply:
This should really be `if path.startswith(self.disallowed_paths)`, `allowed_paths` is confusing because it's clearly a blacklist.

There's one more `disallowed_paths`, it would be prudent to move that to a similar setting for consistency.

Reply:
actually think this *is* allowed_paths..

its basically saying "if the path does not startwith [this list], exit"

Reply:
Oh, oops. You're right.